src/index.js

```diff
@@ -75,7 +68,11 @@ export default function profileStore(options = {}) {
 			performance.clearMarks(markName + START)
 			performance.clearMarks(markName + END)
 			performance.clearMeasures(getMarkLabel(markInfo, markType))
-			counter -= counterDelta
+			if (counter[markInfo.name]) {
+				counter[markInfo.name]--
+			} else {
+				delete counter[markInfo.name]
+			}
```

Comment:
I think there is a bug here, property will be deleted only in the next run. Shouldn't this be:
```suggestion
			counter[markInfo.name]--
			if (!counter[markInfo.name]) {
				delete counter[markInfo.name]
			}
```